src/controllers/translations/retrieve.js

```diff
@@ -1,17 +1,17 @@
 const text = require('src/controllers/text')
 const mongo = require('src/databases/mongo')
-const NotFoundError = require('src/errors/not-found')
 
 module.exports = async (language, signed) => {
+  const translationData = { signed, spoken: null }
   const signedProcessedText = await text.process(signed)
 
   const translation = await retrieveData(language, 'translations', { signed: signedProcessedText.hash })
 
-  if (!translation) throw new NotFoundError()
```

Comment:
It's important to keep this validation error.

Reply:
In this case, shouldn't we just return no translation and let the backend handle this and show an alert to the user that no translation was found?

As our database is still small, most phrases will return no translation.

Reply:
This is the backend repo and `NotFoundError` is equal to no translation was found.